tritium-metrics/src/main/java/com/palantir/tritium/metrics/InstrumentedSslServerSocketFactory.java

```diff
@@ -53,21 +54,25 @@
     }
 
     @Override
+    @MustBeClosed
```

Comment:
Perhaps we should disable these on overrides unless the overridden method is also annotated with `@MustBeClosed`?

Reply:
Yeah, that would help avoid some noise as this rolls out, then we can ratchet it up.

Reply:
fixed by https://github.com/palantir/gradle-baseline/pull/1685